src/vs/workbench/services/search/common/textSearchManager.ts

```diff
@@ -213,7 +214,8 @@ export class TextSearchManager {
 			maxResults: this.query.maxResults ?? Number.MAX_SAFE_INTEGER,
 			previewOptions: this.query.previewOptions,
 			afterContext: this.query.afterContext,
-			beforeContext: this.query.beforeContext
+			beforeContext: this.query.beforeContext,
+			threads: 123// TODO,
```

Comment:
Perhaps assigning this to undefined should be sufficient?

Reply:
I removed the property from here, it seems it will be assigned in the constructor of `RipgrepTextSearchEngine` .